core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceResponseConverterTest.java

```diff
@@ -678,6 +719,29 @@ public void customizedHttpResponse() {
                     .isEqualTo("annotated-value");
         });
 
+        res = aggregated(client.get("/http-file/expect-custom-header"));
```

Comment:
This test case method is a bit long. Could you add a dedicated test method for this?

Reply:
Sure, I will!